### ISSUE TITLE
Unable to open the excel file

### ISSUE THREAD

**Issue by TohWei** [Depechie/XamarinFormsOffice#1]
Hi, I got the source code and run on physical Android device. 
Click on the Export List to excel button throw error: 
Load the Excel app, but then show this error message: 
Can't Open file.
Something's wrong.

**Comment by Depechie** [comment]
Excel is installed I guess?

**Comment by TohWei** [comment]
Yes. It has Excel installed. It is on Android ver 8.0.0, Samsung Galaxy S8.

**Comment by Depechie** [comment]
I need to retest the repo to see what could be wrong.
Did you update any nuget packages?

**Comment by TohWei** [comment]
I read something from here:
https://github.com/OfficeDev/Open-XML-SDK/issues/221
After few attempts, I tried update the System.IO.Packaging.  But still no luck.

**Comment by Depechie** [comment]
Like I said need to retest this... will get back when I have found some time to look at it.

**Comment by TohWei** [comment]
Latest update,... 
As far as what I understand from the source code, by using the code below will open the excel doc in Excel App. However, it does work at all. It just failed to open it. 
Intent intent = new Intent(Intent.ActionView); 

So, instead of using Intent.ActionView, when I replace it with:
new Intent(Intent.ActionSend);

Somehow I manage to attach/share the created excel file via other program.

**Comment by Depechie** [comment]
Weird... still have not found time to retest this. But I know it worked when I created the test app.
Will see if I can verify this weekend.
